docs/resources/cluster.md

```diff
@@ -28,6 +28,7 @@ A RisingWave Cluster
 ### Read-Only
 
 - `id` (String) The NsID (namespace id) of the cluster.
+- `tier` (String) The tier of your RisingWave cluster. When creating a new cluster, the value is `standard`.
```

Comment:
We should list the available options public users can choose or provide a doc link to the options if it exists.

Reply:
Discussed offline, resolve the comment as this is read only attribute.